.github/workflows/test-linux.yml

```diff
@@ -77,31 +77,35 @@ jobs:
   azure-tests:
     runs-on: ubuntu-latest
     needs: [ 'format', 'tests' ]
-    if: ${{ false }}
-    #if: |
-    #  github.event_name != 'pull_request'
-    #  || github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
+    if: |
```

Comment:
Re-enabling the test again because I was able to verify it works again when increasing the timeout in https://github.com/elastic/apm-agent-dotnet/pull/2385

Reply:
cc @Mpdreamz